src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -244,6 +244,10 @@ const ViewPopup = (() => {
             containerSetText('trade_details_profit_loss', '-', { class: 'loss' });
         }
 
+        if (!contract.sell_price && is_ended) {
```

Comment:
I would rather update the above if else to also include this third condition, rather than override it at the end. something like

```
if (!is_ended && final_price) {
...
} else if (is_ended && !contract.sell_price) {
...
} else {
...
}
```